Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -192,8 +192,12 @@ export default class LinearTimelineComponent extends Vue {
         return result;
     }
 
-    private getHeadingDate(date: Date): string {
-        return moment(date).format("ll");
+    private getHeadingDate(dateGroup: DateGroup): string {
+        const dateToDisplay =
```

Comment:
not sure this is necessary, the dateGroup object is guaranteed to have a date member. If the date is invalid then the issue is somewhere else.
Have you checked this section?
```
let groupArrays = Object.keys(groups).map<DateGroup>((dateKey) => {
            return {
                key: dateKey,
                date: new Date(groups[dateKey][0].date),
                entries: groups[
                    dateKey
                ].sort((a: TimelineEntry, b: TimelineEntry) =>
                    a.type > b.type ? 1 : a.type < b.type ? -1 : 0
                ),
            };
        });

Reply:
I believe the date is not correctly calculated due to the "const date = new Date(entry.date).setHours(0, 0, 0, 0);" code. 
Per the screenshot on my comment on https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/8819, this date field is not the original date of the entry and it seems it is misinterpreted on mobile's browsers.

Reply:
Ah, so it could be a timezone issue on that line. If that's the case the solution would also be on that same since that seems to be the root cause of the issue

Reply:
After thinking more about this, the code changes here would affect the display of the data coming from the services. If that would be incorrect then all the dates would be potentially wrong given the timezone. Since this is something that is only happening on creating a new note (the original issue) I presume the error is happening somewhere else

Reply:
Per our discussion, it makes more sense to fix the date grouping issue as well.

Reply:
Few interesting things have been identified as follows:
-  Issue is actually related to the browser timezone and the "return moment(date).format("ll");" (not the date group)
- For debugging this issue in Chrome, we can install the "Change Timezone (Time Shift)" chrome extension.
- The fix has been implemented and successfully tested.